apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -43,7 +46,28 @@ const Auth: React.FC = () => {
 		return null;
 	};
 
-	return <>{getScreen(pathname)}</>;
+	return (
+		<>
+			<main className={styles["auth-container"]}>
+				<section className={styles["form-container"]}>
+					{getScreen(pathname)}
+				</section>
+				<section className={styles["ilustration-container"]}>
```

Comment:
```suggestion
				<section className={styles["illustration-container"]}>
```
Fix for all. You can use spell checker to avoid such mistakes